portfolio/src/main/webapp/index.html

```diff
@@ -23,18 +23,20 @@
       <input type="submit" class="submit" value="Show"></input>
     </form>
     <form id="sortComments">
-      <label for="sortingMethods">Sort:</label>
-      <select id="sortingMethods">
-        <option value="time">Time (newest)</option>
-        <option value="likes">Time (oldest)</option>
+      <label for="sortMethod">Sort:</label>
+      <select id="sortMethod" onchange="applySortMethod()">
+        <option value="DESCENDING" selected="selected">Time (newest)</option>
```

Comment:
selected="selected" is some old XHTML voodoo.  Just selected is fine.  But it's so ingrained that you'll still see people writing ="selected".

Reply:
noted